examples/Folders/Readme.md

```diff
@@ -0,0 +1,49 @@
+Dropzone that accepts folders as drag-and-drop.
+
+```
+const {getDroppedOrSelectedFiles} = require('html5-file-selector')
+
+class FolderDropzone extends React.Component {
+  constructor() {
+    super()
+    this.state = { files: [] }
+  }
+
+  onDrop(files) {
+    this.setState({
+      files
+    })
+  }
+
+  render() {
+    return (
+      <section>
+        <div className="dropzone">
+          <Dropzone
+            getDataTransferItems={evt =>
+              getDroppedOrSelectedFiles(evt).then(list =>
```

Comment:
Hmm, why is the `map` needed? Is this something coming from the plugin?

Reply:
@okonet No, you are right it's redundant